Controller/SubscriptionWallChecker.php

```diff
@@ -35,34 +36,33 @@ class SubscriptionWallChecker
     protected $channel;
 
     /**
-     * @var TokenStorageInterface
+     * @var EntityManager
      */
-    protected $token;
+    protected $em;
 
     /**
-     * @var EntityManager
+     * @var AuthorizationChecker
      */
-    protected $em;
+    protected $ac;
```

Comment:
Following the Symfony code standards (http://symfony.com/doc/current/contributing/code/standards.html) you will end up with somehing like $authorizationChecker instead of ac.